content/inheritance/reading/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Reading"
+date: 2023-02-31T15:39:38-06:00
+draft: false
+weight: 1
+originalAuthor: Courtney Frey # to be set by page creator
+originalAuthorGitHub: speudusa # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
```

Comment:
There are no links.

Reply:
fixed. added key terms and concepts too.

Reply:
Thanks.